cpg_utils/hail_batch.py

```diff
@@ -361,14 +377,54 @@ def reference_path(suffix: str) -> Union[CloudPath, Path]:
     Parameters
     ----------
     suffix : str
-        Describes path relative to the reference prefix.
+        Describes path relative to the reference prefix. Can be a key in the
+        `references` config section.
+    section : str, optional
+        Subsection name inside the references config section.
 
     Returns
     -------
     str
     """
+    prefix = to_path(get_config()['workflow']['reference_prefix'])
+
+    if d := get_config().get('references'):
+        if section:
+            if section not in d:
+                raise ValueError(
+                    f'No subsection {section} in the "references" config section'
+                )
+            d = d[section]
+            if extra_prefix := d.get('prefix'):
+                prefix /= extra_prefix
+        if suffix and suffix in d:
+            suffix = d[suffix]
+
     # A leading slash results in the prefix being ignored, therefore use strip below.
-    return to_anypath(get_config()['workflow']['reference_prefix']) / suffix.strip('/')
+    suffix = suffix.strip('/')
+    return prefix / suffix
+
+
+def genome_build() -> str:
+    """Return the genome build name"""
+    return get_config()['references'].get('genome_build', 'GRCh38')
+
+
+def fasta_res_group(b, indices: list | None = None):
```

Comment:
Should this have "reference" in the method name to make it more clear what this is used for?

Reply:
I don't know, I think I'd leave it.

Sorry for pushing back, but that's a good example of a trivial comment/suggestion that I'd love if we try to avoid in PRs!

Reply:
Of course you're free to disregard the suggestion, but I think commenting on unclear naming isn't something that we should refrain from in PRs.

Just reading this function name (imagine you're new to this module!), it's pretty unclear what this is used for.